### PR TITLE
Add Hashimoto DTT recap

### PR DESCRIPTION
**Pull request by csbailey5t**
A couple of notes:

I'm not sure of the title quote. More of the show is around the iPad and NixOS, so a quote related there might be more appropriate. 

I used two blockquotes in a couple of places. Happy to reduce that or move them around or add more. 

Thanks in advance for any suggestions on what to flesh out, take out, etc.!

### CODE REVIEW

File: blogposts/2021/dev-tool-time-mitchell-hashimoto.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 title: "'I don't like things that write code for me': Dev Tool Time with Mitchell Hashimoto"
-description: 'Mitchell Hashimoto, co-founder, CTO, and now individual contributor at HashiCorp, shares how reproducible environments with NixOS and an iPad fit into his developer workflow.'
+description: 'Mitchell Hashimoto, co-founder, CTO, and now individual contributor at HashiCorp, shares how reproducible environments with NixOS and an iPad fit into his developer workflow, and his sentiments about IDEs.'
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-mitchell-hashimoto.md:3]: @csbailey5t I think they spend enough time on the Vim conversation that the quote works, I just adjusted the description to reference it too though

File: blogposts/2021/dev-tool-time-mitchell-hashimoto.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 author: Scott Bailey
 authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
 publishDate: 2021-08-25T10:00-07:00
 tags: [blog]
 slug: dev-tool-time-mitchell-hashimoto
 heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/dev-tool-time-mitchell-hashimoto.jpg
 socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/dev-tool-time-mitchell-hashimoto.jpg
-published: false
+published: true
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-mitchell-hashimoto.md:11]: Doing this so the Netlify preview works for us 😁

File: blogposts/2021/dev-tool-time-mitchell-hashimoto.md [hunk 3]
```diff
@@ -37,39 +37,43 @@ Mitchell also has a stack of [Raspberry Pi](https://www.raspberrypi.org/) comput
...
+Mitchell uses his iPad and Apple Pencil for a mix of personal and professional work. As a hobbyist pilot, he plots and checks flight paths. As an engineer, he uses the iPad for reading and reviewing code and long-form documents, such as RFCs, and writing handwritten notes with the Apple Pencil and [Goodnotes](https://www.goodnotes.com/). Goodnotes makes it easy to move text around, and reorganize content visually. When he's ready for long-form typing or document creation, Mitchell transfers his notes to his computer with Goodnotes for MacOS and iCloud before drafting in Google Docs, GitHub, or the appropriate app. This setup frees Mitchell from his desk, letting him do code reviews outside.
 
 ## [NixOS](https://nixos.org/)
 
 > "I think one thing that always scared me every single year was the MacOS release and like, what would that break?"
 
+Operating system and other software updates regularly break development environments. Mitchell avoids this recurring problem using NixOS in a VM running with [VMware Fusion](https://www.vmware.com/products/fusion.html).
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-mitchell-hashimoto.md:46]: I reordered this section a bit to break up the pull quotes

File: blogposts/2021/dev-tool-time-mitchell-hashimoto.md [hunk 4]
```diff
@@ -37,39 +37,43 @@ Mitchell also has a stack of [Raspberry Pi](https://www.raspberrypi.org/) comput
...
 - [nix-shell](https://nixos.org/manual/nix/stable/#sec-nix-shell) enables a single-session shell with specific dependencies
-- [direnv](direnv) provides a consistent approach for setting and accessing environment variables based on directory
+- [direnv](https://direnv.net/) provides a consistent approach for setting and accessing environment variables based on directory
 
 ## [Neovim](https://neovim.io/)
 
-> "I just don't feel like I personally need an IDE experience."
-
-> "I don't like things that write code for me; I want to do it myself."
+> "I used to be an Emacs user ... and then I joined this other company and I was harassed on a daily basis about my Emacs use because I was the only Emacs user in the entire company ... I was the butt of every joke! I got so fed up with it I decided to use Vim for one month and I never switched back."
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-mitchell-hashimoto.md:59]: This felt like a more colourful quote so I led with this, and combined the original two pull quotes with ellipses, let me know if this works for you!

**Review by rebeccadee — APPROVED** [review]
@csbailey5t Thanks so much for the quick turnaround on this! I left a couple of comments as FYIs, let me know if all is good and I can merge!

**Comment by csbailey5t** [comment]
@rebeccadee Everything looks great - thanks for the improvements!

**Pull request merged**